polkadot-parachains/integritee-runtime/src/weights/frame_system.rs

```diff
@@ -30,42 +29,41 @@ use sp_std::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
 	fn remark(b: u32, ) -> Weight {
-		(1_783_000 as Weight)
+		(0 as Weight)
```

Comment:
that one frequently goes wrong. shouldn't be zero. I suggest to just leave the old value